opencti-platform/opencti-graphql/src/resolvers/settings.js

```diff
@@ -38,6 +39,7 @@ const settingsResolvers = {
     relationships: (_, __, context) => elAggregationCount(context, context.user, READ_DATA_INDICES, { types: ['stix-relationship'], field: 'entity_type' }),
   },
   Settings: {
+    platform_type: () => (PLATFORM_VERSION.endsWith('lts') ? 'lts' : 'standard'),
```

Comment:
so trial and nfl (or i don't remember the name for Charity licenses) are all standard ? Or it is possible to have trial on lts too ?

Reply:
Its not the type of the license but the type of the platform, so only LTS or STANDARD